src/client/components/navigation-button/navigation-button.style.css

```
@@ -1,21 +1,21 @@
 .icon {
+  color: var(--dark-grey);
```

it should be var(--dark-blue)
```suggestion
  color: var(--dark-blue);
```